plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/StagingDelegate.java

```diff
@@ -52,4 +54,10 @@ IStatus stage(IProject project, IPath stagingDirectory,
    */
   IPath getOptionalConfigurationFilesDirectory();
 
+  /**
+   * Does not guarantee cancellation/termination. This can be called concurrently from a different
+   * thread than the thread running {@link #stage}.
+   */
+  void interrupt();
```

Comment:
Uh weird!  I'm sure what I saw was a class with an empty method body.  Of course you're right.